core/src/term/mod.rs

```diff
@@ -209,6 +209,9 @@ pub enum Term {
     #[serde(skip)]
     Import { path: OsString, format: InputFormat },
 
+    #[serde(skip)]
+    ImportPkg(Ident),
```

Comment:
Although we are getting rid of `Term`, adding a new variant impacts quite a few places with pattern matching where we don't care about imports. Should we just rename `PathOrPackage` to something like `Import`, taking an owned `OsString`, and use that type as the parameter of `Term::Import`? After all the'yre all imports. What do you think?

Reply:
Yep, that turned out nicer.